torch/fx/experimental/proxy_tensor.py

```diff
@@ -266,8 +271,7 @@ def can_handle_tensor(x):
         # this can happen
         and pytree.tree_all_only((SymInt, SymFloat, SymBool), lambda _: False, (args, kwargs))
     )
-
-    if torch.Tag.data_dependent_output in func.tags:  # type: ignore[attr-defined]
+    if not external_call and torch.Tag.data_dependent_output in func.tags:  # type: ignore[attr-defined]
```

Comment:
If `not external` is True, this function already returned at line 256 or 260. Why check it again?

Reply:
No, returns in line 256 only if it is in `CURRENT_DECOMPOSITION_TABLE` etc.

Reply:
Btw `not external` is the old code path, so it's there for a reason.

Reply:
I see. Basically, after line 261, the `func` must be a primitive (defined by decomposition table) or an external function. Thanks.